docs/includes/dagger-cloud-cta.md

```diff
@@ -0,0 +1,6 @@
+## Dagger Cloud
+
+<a href="https://dagger.cloud" target="_blank" rel="">Dagger Cloud</a> is a managed service that aims to help you troubleshoot your current dagger runs by storing operation history and enabling functionality such as storing outputs and a comprehensive detailed view of your executions.
+
+If you're interested in trying out Dagger Cloud, you can find more information in our [docs section](https://docs.dagger.io/1241/dagger-cloud/). Furthermore, if you have any feedback or ideas that could help improve the product, there is an
+[open discussion](https://lala.com) in GitHub where you can leave us your inputs.
```

Comment:
lines 5 & 6 should be one line.

need link to the discussion instead of placeholder lala.com

Reply:
Apparently the a tag is breaking the rendering. When Docusaurus parses the Markdown file it does not create a p element, as it does in the second paragraph.

Reply:
> lines 5 & 6 should be one line.

fixed.

> lines 5 & 6 should be one line.
> 
> need link to the discussion instead of placeholder lala.com
I'll update this today